reporter/html.js

```diff
@@ -467,7 +479,7 @@ function appendUserAgent() {
 		userAgent.innerHTML = "";
 		userAgent.appendChild(
 			document.createTextNode(
-				"QUnit " + QUnit.version  + "; " + navigator.userAgent
+				"QUnit " + QUnit.version	+ "; " + navigator.userAgent
```

Comment:
What is this change? GitHub displays the green "added" diff as unselectable, weird.